### PR TITLE
scripts/runfabtests.sh: Fix multinode test running on OSX devices

### PR DESCRIPTION
**Pull request by AMcKinley15**
The issue of the OSX tests timing out was likely caused by not waiting for client processes to finish executing before killing the server. The bash script now waits for all of those clients to finish.

### CODE REVIEW

File: fabtests/scripts/runfabtests.sh [hunk 1]
```diff
@@ -532,7 +532,7 @@ function multinode_test {
 	s_cmd="${BIN_PATH}${test_exe} ${S_ARGS} -s ${S_INTERFACE}"
 	${SERVER_CMD} "${EXPORT_ENV} $s_cmd" &> $s_outp &
 	s_pid=$!
-	sleep 1
+	sleep 5
```
Comment (shefty) [inline fabtests/scripts/runfabtests.sh]: This change is unrelated to the other change.  1 concept per patch please.

File: fabtests/multinode/src/harness.c [hunk 2]
```diff
@@ -184,7 +184,7 @@ static int pm_conn_setup()
 	int sock,  ret;
 	int optval = 1;
 
-	sock = socket(AF_INET, SOCK_STREAM, 0);
+	sock = socket(pm_job.oob_server_addr.ss_family, SOCK_STREAM, 0);
```
Comment (shefty) [inline fabtests/multinode/src/harness.c]: Please update the commit message to something like:
```
multinode: Support non-AF_INET socket domains

Replace hardcoded AF_INET socket domains with the value obtained from getaddrinfo.
```

File: fabtests/man/man7/fabtests.7 [hunk 3]
```diff
@@ -307,6 +307,12 @@ Tests memory registration.
 Allocates and closes fabric resources to check for proper cleanup.
 .RS
 .RE
+.TP
+.b \f[I]fi_multinode\f[]
+Runs a full mesh, all to one, one to all, and ring patterns on multiple 
+procceses over sockets, rma, atomics, and tagged messages.
```
Comment (shefty) [inline fabtests/man/man7/fabtests.7]: Let's call this documentation lite lite...  Can we get a more complete description of what this test does and how to use it?

File: fabtests/scripts/runfabtests.sh [hunk 4]
```diff
@@ -532,7 +532,7 @@ function multinode_test {
 	s_cmd="${BIN_PATH}${test_exe} ${S_ARGS} -s ${S_INTERFACE}"
 	${SERVER_CMD} "${EXPORT_ENV} $s_cmd" &> $s_outp &
 	s_pid=$!
-	sleep 5
+	sleep 1
```
Comment (shefty) [inline fabtests/scripts/runfabtests.sh]: This change reverts the change from the other commit?

Comment (AMcKinley15) [reply fabtests/scripts/runfabtests.sh]: Yeah this was a guess attempt at a fix and I guess I undid the change incorrectly in git. I'll see if I can change it so that this non-change doesn't show up in the log at all

File: fabtests/man/man7/fabtests.7 (hunk 3 above)
Comment (AMcKinley15) [reply fabtests/man/man7/fabtests.7]: I did this in the md file. Do you also want me to add the same information here?

Comment (shefty) [reply fabtests/man/man7/fabtests.7]: Here we can have a simple explanation, such as:
```
Performs a series of data transfers between 2 or more nodes.  Used for validating at scale.
```

Comment (AMcKinley15) [reply fabtests/man/man7/fabtests.7]: oh, I guess I'll just take this out then

File: fabtests/man/fabtests.7.md [hunk 5]
```diff
@@ -357,6 +357,14 @@ This will run "fi_rdm_atomic" for all atomic operations with
 	- 1024 bytes message size
 	- server node as 123.168.0.123
 
+## Run multinode tests
+
+	server and clients are invoked with the same command: 
+		fi_multinode -n <number of processes> -s <server_addr> 
+	
+	a process on the server must be started before any of the clients can be started 
+	succesfully.
```
Comment (shefty) [inline fabtests/man/fabtests.7.md]: For the multinode test, we want a more complete description of what this does.  See the ubertest description as more as a guide.  We want to explain what the test is doing in more detail.

**Comment by AMcKinley15** [comment]
Should be good to merge at this point.

File: fabtests/man/man7/fabtests.7 [hunk 6]
```diff
@@ -304,7 +304,7 @@ Tests memory registration.
 .RE
 .TP
 .B \f[I]fi_resource_freeing\f[]
-Allocates and closes fabric resources to check for proper cleanup.
+Allocates and closes fabric resources to check for proper cleanup.
```
Comment (aingerson) [inline fabtests/man/man7/fabtests.7]: This change doth annoyeth me.

Comment (aingerson) [inline fabtests/man/man7/fabtests.7]: Wrong patch. Still here.

Comment (shefty) [reply fabtests/man/man7/fabtests.7]: I'm going to go ahead and merge this series even though this change is wrong and removed in a separate patch.  But only because a daemon is going to blast over this man page anyway, as only md pages should ever be updated directly.
